packages/@angular/cli/commands/e2e.ts

```diff
@@ -14,6 +14,7 @@ export interface E2eTaskOptions extends ServeTaskOptions {
   webdriverUpdate: boolean;
   specs: string[];
   elementExplorer: boolean;
+  useDist: boolean;
```

Comment:
`dist` has no special meaning in the CLI terms. It's called `outDir` in the config, and `outputPath` in the `build` command arguments.